docs/articles/infrastructure/ci-cd/freeipa.rst

```diff
@@ -316,4 +490,76 @@ Try to identify an LDAP user:
     # Should return "no such user"
     id someldapuser
 
+Replica
+^^^^^^^
+
+If any of the steps fails during installation you need to run 
+``ipa-server-install --uninstall`` a few times to clean the system. 2-3 should
+do it. After that you need to re-enroll the client before continuing with
```

Comment:
This is awkward. Why does it need to be uninstalled 2-3 times?

Reply:
I understand that since this is a how-to it's good to include this type of troubleshooting info, but I agree that something about the "it often fails and you need to uninstall many times to fix it" phrasing feels a bit off anyway. I don't have any good ideas on how to rephrase it though.

Reply:
Tried to give a reasonable explanation as to why. I can't change the underlying cause tho.